OpenRA.Mods.Common/Traits/BlocksProjectiles.cs

```diff
@@ -13,6 +13,11 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	public interface IBlocksProjectiles
+	{
+		WDist GetHeight(Actor self);
```

Comment:
I think this could be named better, but this is probably in line with the current projectile code.